homeassistant/components/air_quality/norway_air.py

```diff
@@ -0,0 +1,153 @@
+"""
+Sensor for checking the air quality forecast around Norway.
+
+For more details about this platform, please refer to the documentation at
+https://home-assistant.io/components/air_quality.norway_air/
+"""
+import logging
+
+from datetime import timedelta
+import voluptuous as vol
+
+import homeassistant.helpers.config_validation as cv
+from homeassistant.components.air_quality import (
+    PLATFORM_SCHEMA, AirQualityEntity)
+from homeassistant.const import (CONF_LATITUDE, CONF_LONGITUDE,
+                                 CONF_NAME,
+                                 CONF_SHOW_ON_MAP)
+from homeassistant.helpers.aiohttp_client import async_get_clientsession
+
+
+REQUIREMENTS = ['pyMetno==0.4.5']
+
+_LOGGER = logging.getLogger(__name__)
+
+ATTRIBUTION = "Air quality from " \
+              "https://luftkvalitet.miljostatus.no/, " \
+              "delivered by the Norwegian Meteorological Institute."
+# https://api.met.no/license_data.html
+
+CONF_FORECAST = 'forecast'
+
+DEFAULT_FORECAST = 0
+DEFAULT_NAME = 'Air quality Norway'
+
+PLATFORM_SCHEMA = PLATFORM_SCHEMA.extend({
+    vol.Optional(CONF_FORECAST, default=DEFAULT_FORECAST): vol.Coerce(int),
+    vol.Optional(CONF_LATITUDE): cv.latitude,
+    vol.Optional(CONF_LONGITUDE): cv.longitude,
+    vol.Optional(CONF_NAME, default=DEFAULT_NAME): cv.string,
+    vol.Optional(CONF_SHOW_ON_MAP, default=True): cv.boolean,
```

Comment:
Remove this feature please. This is a frontend concern and shouldn't be part of the backend.